Agreements/Internal/TCRED_TCDAG.txt

```diff
@@ -0,0 +1,3 @@
+COPX:*:*:LAPRA:EGSS:*:London TC REDFA:London TC DAGGA:*:12000:LAPRA
```

Comment:
See comment on the TC ABBOT Stack file below. I think it makes more sense for Redfa to have responsibility for the stack, so these agreements could be removed or saved for when/if we have a TC Lorel sector that looks more like the real world's.